### PR TITLE
daily validation tasks

### LINKED ISSUE #84: add daily observation validation task

### PR DESCRIPTION
**Pull request by alorenzo175**
closes #84

### CODE REVIEW

File: solarforecastarbiter/io/api.py [hunk 1]
```diff
@@ -15,6 +16,18 @@
...
+logger = logging.getLogger(__name__)
+
+
+def request_cli_access_token(user, password):
+    req = requests.post(
+        'https://solarforecastarbiter.auth0.com/oauth/token',
+        data={'grant_type': 'password', 'username': user,
+              'audience': BASE_URL,
+              'password': password,
+              'client_id': 'c16EJo48lbTCQEhqSztGGlmxxxmZ4zX7'})
```
Comment (wholmgren) [inline solarforecastarbiter/io/api.py]: I'm guessing this corresponds to something meaningful...

File: solarforecastarbiter/validation/tasks.py [hunk 2]
```diff
@@ -280,3 +371,70 @@ def immediate_observation_validation(access_token, observation_id, start, end,
...
+    'dc_power': validate_daily_dc_power,
+    'ac_power': validate_daily_ac_power
+}
+
+
+def _daily_validation(session, observation, start, end, base_url):
+    observation_values = session.get_observation_values(
+        observation.observation_id, start, end)
+    value_series = observation_values['value']
+    if len(value_series.dropna()) < 10:
```
Comment (wholmgren) [inline solarforecastarbiter/validation/tasks.py]: How did you decide on 10? 

Hourly data in the winter might have less than 10 data points if only uploaded during the day. Maybe it shouldn't work in that case, just pointing it out.

File: solarforecastarbiter/io/api.py (hunk 1 above)
Comment (alorenzo175) [reply solarforecastarbiter/io/api.py]: yes, but not something private

File: solarforecastarbiter/validation/tasks.py (hunk 2 above)
Comment (alorenzo175) [reply solarforecastarbiter/validation/tasks.py]: From https://github.com/SolarArbiter/solarforecastarbiter-core/issues/71#issuecomment-486274741

>Checks requiring a time series: detect_stale_values, detect_interpolation, detect_clipping. For these, I suggest requiring at least 24 hours(?) of data? Or 10+ data points. With shorter time series the code should run but the output is probably not useful.

The other checks should already have been done by the immediate validation, so OK to fail when < 10

Comment (alorenzo175) [reply solarforecastarbiter/validation/tasks.py]: Since the functions are windows based on count, I figured I would go with 10 vs 24 hrs

**Pull request merged**